internal/cmd/common.go

```diff
@@ -19,6 +20,8 @@ import (
 	"github.com/stateful/runme/internal/runner/client"
 )
 
+const envStackDepth = "__RUNME_STACK_DEPTH"
```

Comment:
My reasoning is that I wanted to strongly emphasize that this is "private," or in other words not intended for users. Especially since, theoretically, tampering with this value can result in bad consequences (infinite recursion).

This would be in contrast with something like, for example, `$RUNME_PREVIOUS_OUTPUT`, which might be intended for users' usage.

I don't care much either way though. If you think it's better to have consistency I'm fine with that.

Reply:
Internal diagnostics? I'd imagine this ENV var isn't meant to be a "public interface".

Reply:
Ah ok, this tracks the current stack depth which definitely would not be considered to be used by users. I wonder if we want to make this configurable 🤔 e.g. allowing power user to increase the depth. `10` seems to be high enough for now though I would be suprised if someone goes higher than that

Reply:
I just ran it with the panic conditional removed. My computer's vitals didn't break a sweat well over 300. How about `100` as the default for now? Put a `// todo` comment in there to consider making it configurable closer to flipping opt-in to default behavior?